daac/variables/sbx.tfvars

```diff
@@ -1,6 +1,9 @@
-#workflow_bucket_names = []
-#standard_bucket_names = ["private", "dashboard"]
-#partner_bucket_names = ["non-prefixed-external-bucket"]
+# Set the maturity specific part of your bucket configuration here.
+# bucket_config = {
+#   "acme-browse"     = { oai = "ABCDEFG1234567" }
```

Comment:
just confirming what I think this can do. If I you want a type public with oai its as simple as:
```
"my-cool-test-bucket" = { oai = "ABCDEFG1234567", type = "public" }
```

Reply:
Basically yes except that the bucket type is special and needs to be set in the base config rather than the per maturity overrides. This is mostly just there as a sanity check since I don't think there is a use case for setting a bucket to a different type for different maturities.

> Trying to override bucket type in maturity isn't allowed. Base bucket type is used.